apps/connect/source/ftrack_connect/plugin_manager/__init__.py

```diff
@@ -273,13 +318,25 @@ def _update_overlay(self, item):
             f'{item.data(ROLES.PLUGIN_NAME)}, Version {str(item.data(ROLES.PLUGIN_VERSION))}'
         )
 
+    def _get_incompatible_plugin_names(self):
+        result = []
+        for plugin in self.installed_plugins:
+            if plugin['incompatible']:
+                result.append(plugin['name'])
+        return result
+
+    def _get_deprecated_plugin_names(self):
+        result = []
+        for plugin in self.installed_plugins:
+            if plugin['deprecated']:
+                result.append(plugin['name'])
+        return result
+
     def _on_apply_changes(self):
         '''User wants to apply the updates, warn about conflicting plugins.'''
-        incompatible_plugins = (
-            self._plugin_list_widget.get_incompatible_plugins()
-        )
-        deprecated_plugins = self._plugin_list_widget.get_deprecated_plugins()
-        unloadable_plugins = incompatible_plugins
+        incompatible_plugin_names = self._get_incompatible_plugin_names()
+        deprecated_plugins = self._get_deprecated_plugin_names()
+        unloadable_plugins = incompatible_plugin_names
         if unloadable_plugins:
```

Comment:
```suggestion
        if incompatible_plugin_names:
```
I can't add the entire suggestion as github doesn't allow that, but remember to update the variable name in lines 346 and 357